text/0000-deprecate-http-tx.md

```diff
@@ -0,0 +1,121 @@
+- Title: Deprecate HTTP(S) Transactions
+- Authors: [joltz](mailto:joltz@protonmail.com)
+- Start date: April 28 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Deprecating HTTP(S) as a transaction method helps preserve privacy for Grin users during the slate exchange process while building transactions. By only allowing the http listener in `grin-wallet` to listen on localhost by default, users and services will be encouraged to use more privacy-friendly transaction methods like Tor or directly exchanging armored slates via the [Slatepack](https://github.com/mimblewimble/grin-rfcs/blob/master/text/0015-slatepack.md) standard. This improves privacy for the Grin ecosystem by ensuring that services do not force Grin users into transaction methods that can easily violate their privacy like HTTP(S).
```

Comment:
I'm not sure about the implementation, is the `By only allowing the http listener in `grin-wallet` to listen on localhost by default` correct? They won't be encouraged, they will be required to use Slatepack

Reply:
Since not all wallet implementations use grin-wallet, we cannot require that all possible wallets, users and services abandon http(s). We can only encourage this by removing support in the scope of grin-wallet. Does that make sense or should I try to rephrase?

Reply:
"we can only encourage this by removing support" make sense to me, anyone could have the freedom of building a custom implementation for http(s), but that implementation is not officially supported, that's what I get, is that what I'm supposed to understand?

Reply:
It's fine on my end :+1: